.github/workflows/power9_tpls.yml

```diff
@@ -0,0 +1,106 @@
+name: github-POWER9-TPLS
+
+# Only allow manual runs until at2 runners are available.
+on: workflow_dispatch
+  #pull_request:
+  #  paths-ignore:
+  #  - '**/*.rst'
+  #  - '**/*.md'
+  #  - '**/requirements.txt'
+  #  - '**/*.py'
+  #  - 'docs/**'
+  #  types: [ opened, reopened, synchronize ]
+
+permissions:
+  contents: none
+
+# Cancels any in progress 'workflow' associated with this PR
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  PR_POWER9_GCC930_OPENMP_SERIAL_LEFT_OPENBLAS_OPENLAPACK_REL:
+    name: PR_POWER9_GCC930_OPENMP_SERIAL_LEFT_OPENBLAS_OPENLAPACK_REL
```

Comment:
That name should probably change similarly to the previous comment about openlapack